test/Microsoft.AspNet.Razor.Test/Framework/ParserTestBase.cs

```diff
@@ -304,6 +304,7 @@ private static void EvaluateSyntaxTreeNode(ErrorCollector collector, SyntaxTreeN
             if (actual == null)
             {
                 AddNullActualError(collector, actual, expected);
+                return;
```

Comment:
This test code was never hit before shockingly. Now that I'm expecting `null` values for some `SyntaxTreeNode`s it was being hit and therefore throwing at line 310.